android/app/src/main/java/com/zulipmobile/MainApplication.java

```diff
@@ -43,25 +36,21 @@ public boolean getUseDeveloperSupport() {
 
         @Override
         protected List<ReactPackage> getPackages() {
-            return Arrays.asList(
-                    new MainReactPackage(),
-                    new AsyncStoragePackage(),
-                    new NetInfoPackage(),
-                    new DocumentPickerPackage(),
-                    new RNCWebViewPackage(),
-                    new RNTextInputResetPackage(),
-                    new ImagePickerPackage(),
-                    new OrientationPackage(),
-                    new RNSentryPackage(),
-                    new PhotoViewPackage(),
-                    new RNFetchBlobPackage(),
-                    new RNSoundPackage(),
-                    new RNDeviceInfo(),
-                    new ZulipNativePackage(),
-                    new NotificationsPackage(),
-                    new SharingPackage(),
+            @SuppressWarnings("UnnecessaryLocalVariable")
+            List<ReactPackage> packages = new PackageList(this).getPackages();
+            // Packages that cannot be autolinked yet can be added
+            // manually here
+            packages.add(new ZulipNativePackage());
+            packages.add(new NotificationsPackage());
+            packages.add(new SharingPackage());
+
+            // Add unimodules
+            List<ReactPackage> unimodules = Arrays.<ReactPackage>asList(
                     new ModuleRegistryAdapter(mModuleRegistryProvider)
             );
+            packages.addAll(unimodules);
```

Comment:
Is this different from just `packages.add(new ModuleRegistryAdapter(…))`?

Reply:
Hmm, interesting! I don't know what else might be supposed to go in this list, eventually. I was following this: https://gist.github.com/mczernek/9de9e184abc430e9e3508d26738c8a14/revisions

which is linked to from https://github.com/unimodules/react-native-unimodules, as "this diff for react-native >= 0.60".

I think it makes sense to just do `packages.add(new ModuleRegistryAdapter(…))`.